project_liberation/main_page/templates/head.haml

```diff
@@ -0,0 +1,101 @@
+%meta{:name => "viewport",
+        :content => "width=device-width, initial-scale=1"}
+
+%link{:rel => "stylesheet",
+		:type => "text/css",
+		:href => "https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/css/bootstrap.css",
+		:integrity => "sha384-t4IGnnWtvYimgcRMiXD2ZD04g28Is9vYsVaHo5LcWWJkoQGmMwGg+QS0mYlhbVv3",
+		:crossorigin => "anonymous"}
+
+%script{:type => "text/javascript",
+		:src => "https://code.jquery.com/jquery-3.4.1.js",
+		:integrity => "sha384-mlceH9HlqLp7GMKHrj5Ara1+LvdTZVMx4S1U43/NxCvAkzIo8WJ0FE7duLel3wVo",
+		:crossorigin => "anonymous"}
+
+%script{:type => "text/javascript",
+		:src => "https://stackpath.bootstrapcdn.com/bootstrap/4.3.1/js/bootstrap.js",
+		:integrity => "sha384-rkSGcquOAzh5YMplX4tcXMuXXwmdF/9eRLkw/gNZG+1zYutPej7fxyVLiOgfoDgi",
+		:crossorigin => "anonymous"}
+
+%script{:type => "text/javascript",
+		:src => "https://unpkg.com/aos@2.3.1/dist/aos.js",
+		:integrity => "sha384-wziAfh6b/qT+3LrqebF9WeK4+J5sehS6FA10J1t3a866kJ/fvU5UwofWnQyzLtwu",
+		:crossorigin => "anonymous"}
+
+%link{:type => "text/javascript",
+		:href => "https://unpkg.com/aos@2.3.1/dist/aos.css", :rel => "stylesheet",
+		:integrity => "sha384-/rJKQnzOkEo+daG0jMjU1IwwY9unxt1NBw3Ef2fmOJ3PW/TfAg2KXVoWwMZQZtw9",
+		:crossorigin => "anonymous"}
```

Comment:
As far as I remember one of good html/haml practices is to link to js script in the bottom of body.